### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by xefoci7612** [karpathy/llama2.c#425]
Maintainer @karpathy is inactive since many weeks, he made [a single commit few days ago](https://github.com/karpathy/llama2.c/commit/d0237abd32e553317a2bd80ecd5d4c621ddd307a), but it seems more like a "ping signal" than a real commit.

We can understand he is very busy and currently does not have time to dedicate to this project.

There are many patches waiting in queue, including sensible ones like [quantization](https://github.com/karpathy/llama2.c/commits/feature/int8_try2), that would allow to run a 7B model without a powerful PC, or [support for tinyllama](https://github.com/karpathy/llama2.c/commits/feature/int8_try2), another work that goes in the same direction of allow hacking on LLM code for people without beefy hardware (BTW this is the initial goal that started this project).

So my question is, would it make sense for @karpathy, if he foresees that his time will be limited, to pick one or more maintainers that can help him to keep this ball rolling?

**Comment by karpathy** [comment]
Yeah I don't have too much time right now for this repo. Please link to any PRs that you consider no-brainers, happy to take a look. Maybe I should merge the quantization branch.

**Comment by xefoci7612** [comment]
> Yeah I don't have too much time right now for this repo. Please link to any PRs that you consider no-brainers, happy to take a look. Maybe I should merge the quantization branch.

@karpathy thx for answering

The quantization branch is really important to be merged IMHO because many patches need to be refactored above it (it changes the model layout), so until is not merged other work is in a kind of state of flux.

**Comment by karpathy** [comment]
I see. It is currently a whole separate file runq.c. Which I don't love, but also don't really see any real way around.
Let me re-load my RAM again with where that PR was again... iirc it was pretty much ready to be merged except it was surprisingly not as fast as I originally expected.

**Comment by xefoci7612** [comment]
> I see. It is currently a whole separate file runq.c. Which I don't love, but also don't really see any real way around. Let me re-load my RAM again with where that PR was again... iirc it was pretty much ready to be merged except it was surprisingly not as fast as I originally expected.

THX for merging.

Regarding speed, maybe it is system dependent, in my case is way faster, like more than 2X faster.

**Comment by karpathy** [comment]
@KangkangStu did you follow instructions here? https://github.com/karpathy/llama2.c#int8-quantization

**Comment by KangkangStu** [comment]
> @KangkangStu您遵循此处的说明了吗？https://github.com/karpathy/llama2.c#int8-quantization
Of course, I tried llama2_7b and it succeeded, but when I wanted to quantify stories15M.pt there was nothing I could do
